writeit/settings.py

```diff
@@ -88,8 +88,11 @@
 # Example: "http://example.com/static/", "http://static.example.com/"
 STATIC_URL = '/static/'
 
+INSTANCE_STATIC = os.path.abspath(os.path.join(BASE_DIR, '..', 'static'))
+
 # Additional locations of static files
 STATICFILES_DIRS = (
+    INSTANCE_STATIC,
```

Comment:
The new `instance` app contains the model that represents a subdomain, so perhaps it would be simpler to say custom CSS for subdomains has to go under: `instance/static/` - then you wouldn't need this change.